tensorflow/python/keras/preprocessing/__init__.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ==============================================================================
-"""Keras data preprocessing utils."""
+"""Provides keras data preprocessing utils to pre-process tf.data.Datasets before they are fed to the model."""
```

Comment:
Is this intended to be used with tf.data.Dataset? IIUC it should be the layers under tensorflow/python/keras/layers/preprocessing

Reply:
```suggestion
"""Provides keras data preprocessing utils to pre-process data fed to the keras layers."""
```
Will this be okay?

Reply:
@tanzhenyu Can you please take a look on the above comment from @ManishAradwad. Thank you!

Reply:
@tanzhenyu Any update on this PR? Please. Thanks!

Reply:
@tanzhenyu Any update on this PR? Please. Thanks!